RiveSharp.Views.UWP/RivePlayer.cs

```diff
@@ -198,6 +200,45 @@ public void FireTrigger(string name)
             EnqueueStateMachineInput(() => mScene.FireTrigger(name));
         }
 
+        public static readonly DependencyProperty sStateMachineInputsProperty = DependencyProperty.Register(
+            nameof(StateMachineInputs),
+            typeof(DependencyObjectCollection),
+            typeof(RivePlayer),
+            new PropertyMetadata(null, new PropertyChangedCallback(OnStateMachineInputsChanged))
+        );
+
+        public DependencyObjectCollection StateMachineInputs
```

Comment:
Thanks for catching that. I made the "StateMachineInputs" class later then forgot to come back and update these!

`new PropertyMetadata(null, new PropertyChangedCallback(OnStateMachineInputsChanged))`

Does that already initialize the property to null?

Could I initialize it to an empty collection through there?

```
    public static readonly DependencyProperty StateMachineInputsProperty = DependencyProperty.Register(
        nameof(StateMachineInputs),
        typeof(StateMachineInputs),
        typeof(RivePlayer),
        new PropertyMetadata(new StateMachineInputs(),
                             new PropertyChangedCallback(OnStateMachineInputsChanged))
    );
```

Reply:
Nevermind. I want to pass "this" to the constructor now, so that won't work here anyway.

Reply:
Yeah, glad you sorted it out, but in general you'd want to usually use `null` or a constant as the default for a dependency property, as otherwise depending on timing of things multiple collection instances could get created.